lte/gateway/c/core/oai/tasks/amf/amf_fsm.cpp

```diff
@@ -117,6 +117,10 @@ void create_state_matrix() {
       REGISTERED_CONNECTED, STATE_EVENT_DEREGISTER, SESSION_NULL,
       DEREGISTERED_INITIATED, SESSION_NULL, "Deregister_Initiated");
 
+  Update_ue_state_matrix(
```

Comment:
nit: missed this previously but why is this specific function beginning with uppercase? we should follow the OAI codebase conventions

Reply:
somehow from the beginning these functions named like this. other functions in this file also needs to be renamed . Will create Zenhub task for this